vpn/src/main/java/com/duckduckgo/mobile/android/vpn/processor/TunPacketReader.kt

```diff
@@ -97,6 +99,9 @@ class TunPacketReader @AssistedInject constructor(
         } else if (packet.isTCP) {
             queues.tcpDeviceToNetwork.offer(packet)
             healthMetricCounter.onWrittenToDeviceToNetworkQueue()
+        } else {
+            healthMetricCounter.onTunUnknownPacketReceived()
+            deviceShieldPixels.sendUnknownPacketProtocol(packet.ip4Header.protocolNum.toInt())
```

Comment:
do we want to send a pixel every time we see an unknown packet or batch them and send them once in a while?